themes/bootstrap3/templates/RecordDriver/DefaultRecord/label-list.phtml

```diff
@@ -0,0 +1,3 @@
+<?php foreach ($this->driver->getLabels() as $label): ?>
+    <span class="label label-default label-<?=$label['class']?>"><?=$this->transEsc($label['label'])?></span>
```

Comment:
Should we be escaping `$label['class']` just to be safe? Also, should we have just a two-space indent here for consistency?

Reply:
Done!